Pos_end:
  guid: 08ba667c-d0e1-11e5-8c18-94c691a73828
  offsets_guid: 08bb445c-d0e1-11e5-8c18-94c691a73828
  orientation_offset: 0.0
  type_id: 11
  x_offset: -1.0
  y_offset: 0.0
delivery_marker:
  guid: 017f3480-fb3d-11ee-947c-94c691a73828
  offsets_guid: 018037c1-fb3d-11ee-947c-94c691a73828
  orientation_offset: 0.876
  type_id: 11
  x_offset: -0.58
  y_offset: 0.024
